var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -65,6 +64,8 @@ class Perl(Package):  # Perl doesn't use Autotools, it should subclass Package
 
     depends_on('gdbm')
     depends_on('berkeley-db')
+    depends_on('bzip2+shared')
+    depends_on('zlib+shared')
```

Comment:
Why `+shared`?

Reply:
My  guess is so you can use the system bzip2 and zlib shared libraries through an external.

Reply:
If the default is shared=True for bzip2 and zlib this is not needed.